torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3185,6 +3230,47 @@ def sample_inputs_aminmax(op_info, device, dtype, requires_grad, **kwargs):
 
     return samples
 
+def error_inputs_aminmax(op_info, device, **kwargs):
+
+    # Error Inputs for zero-dim tensors, when 'dim' arg is not provided.
+    shape = (S, 0, S)
+    err_msg1 = "cannot compute aminmax over an empty dimension as the operation has no identity"
+    yield ErrorInput(SampleInput(torch.rand(shape, device=device)), error_type=RuntimeError,
+                     error_regex=err_msg1)
+
+    # Error Inputs for tensors with more than 64 dimension
+    sizes = [1] * 65
+    err_msg2 = "only tensors with up to 64 dims are supported"
+    yield ErrorInput(SampleInput(torch.randn(sizes, device=device), kwargs={'dim': -1}),
+                     error_type=RuntimeError, error_regex=err_msg2)
+    yield ErrorInput(SampleInput(torch.randn(sizes, device=device), kwargs={'dim': 64}),
+                     error_type=RuntimeError, error_regex=err_msg2)
+
+    # Error Inputs for mixed devices
+    shape = (M)
+    err_msg3 = ""
+    if torch.cuda.is_available():
+        values = torch.randn(shape).cuda()
+        indices = torch.cuda.LongTensor()
```

Comment:
`aminmax` does not produce values and indices, it produces `min` and `max` tensors (both floating point)